src/Components/DetailsPage.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { BsChevronLeft } from 'react-icons/bs';
+import { useSelector, useDispatch } from 'react-redux';
+import './detailsPage.css';
+import { CiSettings } from 'react-icons/ci';
+import { BiMicrophone } from 'react-icons/bi';
+import { fetchCountry } from '../Redux/details';
+
+function Details() {
+  const { country: countryName } = useParams();
+  const dispatch = useDispatch();
+  const country = useSelector((state) => state.country.country);
+
+  useEffect(() => {
+    if (countryName) {
+      dispatch(fetchCountry(countryName));
+    }
+  }, [dispatch, countryName]);
+  return (
+    <div>
+      <header>
+        <li>
+          <Link to="/">
+            <BsChevronLeft style={{ width: '20px', height: '15px' }} className="left-con" />
+            {' '}
+            <h5>2023</h5>
+            {' '}
+          </Link>
+          <BiMicrophone style={{ width: '30px', height: '30px', padding: '10px' }} />
+          {' '}
+          <CiSettings style={{ width: '30px', height: '30px', padding: '10px' }} />
+        </li>
+      </header>
+      {country.name ? (
+        <div>
+
+          <div className="country-title">
+            <h1>{country.name.common.toUpperCase()}</h1>
+            <img src={country.flags.svg} alt={`Flag of ${country.name.common}`} className="Flag" />
+          </div>
+          <ul className="country-details">
+            <li>
+              <h4 className="detail">Region:</h4>
+              {country.region}
+            </li>
+            <li>
+              <h4 className="detail">Capital City:</h4>
+              {' '}
+              {country.capital}
+            </li>
+            <li>
+              <h4 className="detail">Population:</h4>
+              {' '}
+              {country.population}
+            </li>
+            <button type="submit" className="view"><a href="{country.maps.googleMaps}">View on map</a></button>
+          </ul>
+        </div>
+      ) : (
+        <div className="loading">Loading...</div>
+      )}
+    </div>
```

Comment:
Nice and beuatiful design so far, however, as stated in the project requirements, you are suppose to follow the design guidelines including: 

- [x] Colors (select one main color for your website).
- [x] Typography: font face, size and weight.
- [x] Layout: composition and space between elements.

There are some differences between yours and that of the actual design which you will need to fix.

1. The background of each odd card is darker than its preceeding (even) card forming a checkered layout.
2. Also, kindly note the margin, paddings, and distance between the cities name (your green text) and the views (your white text)
3. Your design is also missing the small arrow to the right of each card. 

### Your design 

![Screenshot from 2023-02-09 22-58-58](https://user-images.githubusercontent.com/65233860/217957927-ca2886be-5279-4a86-a24f-2f119faaaee5.png)


### The actual design 

![Screenshot from 2023-02-07 18-50-02](https://user-images.githubusercontent.com/65233860/217342011-f5bc0abd-fc26-4616-bfa8-9c25942c72c9.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: